viewer/packages/camera-manager/src/DefaultCameraManager.ts

```diff
@@ -97,13 +97,15 @@ export class DefaultCameraManager implements CameraManager {
     this._controls = new ComboControls(this._camera, domElement);
     this._controls.minZoomDistance = DefaultCameraManager.DefaultMinZoomDistance;
 
-    this._controls.addEventListener('cameraChange', event => {
-      const { position, target } = event.camera;
-      this._events.cameraChange.fire(position.clone(), target.clone());
-      this._nearAndFarNeedsUpdate = true;
-    });
+    this._controls.addEventListener('cameraChange', this.cameraChange);
   }
 
+  cameraChange = (event: any): void => {
```

Comment:
Use plain functions. Also, should be private? Get rid of `any`.

Reply:
Will be in a new PR